ApplicationInsightsRequestLogging/ApplicationInsightsRequestLogging/Filters/ISensitiveDataFilter.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Azureblue.ApplicationInsights.RequestLogging.Filters
```

Comment:
```suggestion
﻿namespace Azureblue.ApplicationInsights.RequestLogging
```
May I suggest to keep all in a single namespace to shorten the list of `using` statements?